sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -473,6 +530,51 @@
     "all_records"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we want to load data for the added features into the updated customers_feature_group, use `PutRecord` operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session.boto_session.client(\n",
+    "    \"sagemaker-featurestore-runtime\", region_name=region\n",
+    ").put_record(\n",
+    "    FeatureGroupName=customers_feature_group_name, \n",
+    "    Record=[\n",
+    "        {\n",
+    "            \"FeatureName\": \"customer_id\",\n",
+    "            \"ValueAsString\": \"573291\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"FeatureName\": \"EventTime\",\n",
+    "            \"ValueAsString\": str(current_time_sec)\n",
+    "        },\n",
+    "        {\n",
+    "            \"FeatureName\": \"email\",\n",
+    "            \"ValueAsString\": \"john.lee@gmail.com\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"FeatureName\": \"name\",\n",
+    "            \"ValueAsString\": \"John Lee\"\n",
+    "        }        \n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use the `GetRecord` operation again to check the records that have been ingested to the updated customers_feature_group."
```

Comment:
Where is the verification?

Reply:
I will add another cell using GetRecord to validate the above PutRecord is made successful.